systest/cluster/cluster.go

```diff
@@ -248,3 +378,29 @@ func defaultTargetOutbound(size int) int {
 	}
 	return int(0.3 * float64(size))
 }
+
+func persistFlags(ctx *testcontext.Context, config map[string]DeploymentFlag) error {
+	data := map[string]string{}
+	for _, flag := range config {
+		data[flag.Name] = flag.Value
+	}
+	cfgmap := corev1.ConfigMap("flags", ctx.Namespace).
+		WithData(data)
+	_, err := ctx.Client.CoreV1().ConfigMaps(ctx.Namespace).Apply(ctx, cfgmap, metav1.ApplyOptions{FieldManager: "test"})
+	if err != nil {
+		return fmt.Errorf("failed to persist accounts %+v %w", data, err)
+	}
+	return nil
+}
+
+func recoverFlags(ctx *testcontext.Context) (map[string]DeploymentFlag, error) {
+	flags := map[string]DeploymentFlag{}
+	cfgmap, err := ctx.Client.CoreV1().ConfigMaps(ctx.Namespace).Get(ctx, "flags", metav1.GetOptions{})
```

Comment:
curious: metav1.GetOtions{} doesn't to match metav1.ApplyOptions{}. is "test" the default value?

Reply:
FieldManager needs to be set when object is created, it is not relevant for GetOptions